src/openrct2/interface/Viewport.cpp

```diff
@@ -58,10 +57,12 @@ static uint32_t _currentImageType;
 
 static rct_drawpixelinfo _viewportDpi1;
 static rct_drawpixelinfo _viewportDpi2;
-static uint8_t _interactionSpriteType;
-static int16_t _interactionMapX;
-static int16_t _interactionMapY;
-static uint16_t _interactionFlags;
+struct InteractionInfo
+{
+    CoordsXY loc;
+    TileElement* element;
```

Comment:
```suggestion
    TileElement* element { nullptr };
```
Initialize it, to keep the old behavior